service_container/service_decoration.rst

```diff
@@ -225,7 +225,7 @@ automatically changed to ``'.inner'``):
 
 .. note::
 
-    All custom :ref:`service tags </service_container/tags>`_ from the decorated
+    All custom :ref:`service tags </service_container/tags>` from the decorated
```

Comment:
```suggestion
    All custom :doc:`service tags </service_container/tags>` from the decorated
```

Reply:
Yes, this was needed too. I did this while merging. Thanks!